spec/fixtures/deposit/tagmanifest-checksum-mismatch/.nfsd8b8ad4e23fb00930000030c

```diff
@@ -0,0 +1,2 @@
+Tag-File-Character-Encoding: UTF-8
```

Comment:
This is what the offending NFS client temp file looks like in the example linked from the issue this closes.